readme.md

```diff
@@ -5,8 +5,8 @@
 
 ## Install
 
-```
-$ npm install npm-conf
+```bash
+npm install npm-conf
 ```
```

Comment:
Don't do unrelated changes. It's like this for a reason. It's not valid `bash` when using `$`.